### ISSUE TITLE
`battery-stats-collector` should just issue usable lines on stdout and no warning.

### ISSUE THREAD

**Issue by b3** [petterreinholdtsen/battery-stats#23]
In `battery-stats-collector` when `get_logline` is unable to determine a value for `ac_online` it echoes a message warning that "_No power supply found_". This line then goes to the log file before the status line is echoed.

That way the generated file becomes unusable since every two lines it contains non-data. On my host it looks likes something like :

```
No power supply found
1474886747 56.11 1 2016/09/26 12:45:47 -
No power supply found
1474886807 56.45 1 2016/09/26 12:46:47 -
No power supply found
1474886867 56.82 1 2016/09/26 12:47:47 -
```

The warning line should not be echoed, or at least not on stdout.

**Comment by b3** [comment]
I created a branch on my fork which propose a solution to the issue : b3/battery-stats@7ea441d1295abbe83a80323f40a27732d6d52553

**Comment by b3** [comment]
I do not think that the missing power supply should cause a cron email. In fact I am unsure that this message should be raised at all. Without power supply detection `battery-stats-collector` still work.

My idea for the proposition was that if the connected status (online or offline) can be detected it should be logged, but if it can not be detected that information shoud be logged too.

That was why I proposed 3 states.

I do not presuppose any use of that last information though.

**Comment by petterreinholdtsen** [comment]
Right.  make sense to me.

**Comment by b3** [comment]
Do you think my proposition should be integrated in your code? If yes I can just make a pull request of my demo branch.

**Comment by petterreinholdtsen** [comment]
[Bruno BEAUFILS]

> Do you think my proposition should be integrated in your code? If yes
> I can just make a pull request of my demo branch.

Yes.  Please make a pull request.

## 

Happy hacking
Petter Reinholdtsen

**Comment by GillesMocellin** [comment]
I have the same message in logs "No power supply found".
For me, it's because my power supply is seen as ADP0, battery-stats-collector looks for aconline status in several places, including ADP1, but not ADP0.
Removing Warnings will solve the problem and perhaps others.
Adding a look at ADP0 will solve som problems too.
